proprietary/purmerend-design-tokens/figma/figma.tokens.json

```diff
@@ -1481,11 +1481,14 @@
           "fill-1": { "$value": "{purmerend.color.blue-41}" },
           "fill-2": { "$value": "{purmerend.color.blue-41}" },
           "text-1": { "$value": "{purmerend.color.blue-44}" },
-          "text-2": { "$value": "{purmerend.color.blue-44}" }
+          "text-2": { "$value": "{purmerend.color.blue-44}" },
+          "text-3": { "$value": "{purmerend.color.blue-42}" }
         },
         "primary-inverse": {
           "bg-1": { "$value": "{purmerend.color.cyan-6}" },
           "bg-2": { "$value": "{purmerend.color.cyan-6}" },
+          "bg-alert": {"$value": "{purmerend.color.blue-41}"},
```

Comment:
Zou je voor alert een nieuwe scale willen maken, met bijvoorbeeld:

```
"alert": {
  "bg-1": {"$value": "{purmerend.color.blue-41}"},
  "bg-2": {"$value": "{purmerend.color.blue-41}"},
}
```

En klopt het uberhaupt wel dat de Alert-kleur blauw is en niet rood?

Reply:
Van de [Utrecht Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-alert--docs) had ik begrepen dat er verschillende varianten van een Alert Component kunnen zijn. Daarom heb ik een Alert component gekozen om de section "bel/mail de gemeente" op stap 1 beter te laten opvallen. Die is dan blauw, zodat het een info variant is (had ik hier sowieso beter een ander component voor kunnen gebruiken @Robbert ?).

Als het een Alert Component blijft, kan in de tokens misschien onderscheid gemaakt worden tussen alert-info, alert-error etc? (@Marwaxhello )

Reply:
Spotlight section scale ✅